ledger/ledger-runner-common/src/main/scala/com/daml/ledger/runner/common/ConfigAdaptor.scala

```diff
@@ -40,4 +42,14 @@ class ConfigAdaptor {
 
   def authService(participantConfig: ParticipantConfig): AuthService =
     participantConfig.authentication.create(participantConfig.jwtTimestampLeeway)
+
+  def jwtVerifierLoader(
```

Comment:
`CachedJwtVerifierLoader` is part of `ledger-api-auth` module, so I cannot bring `ParticipantConfig` there for example. Also it is part of the instance method so it can be overriden in tests.